Update user guide (#127)
project-chip/certification-tool: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

.. Not having the *$PATH_TO_PAA_ROOTS* set properly when starting the docker or not having PAA roots certificates at that path.
.. Follow the instructions for item 2 in <<prerequisite, Section 8.5.1, Prerequisite>>.

==== Run Tests on the TH User Interface

Some automated Python scripts are available in TH User Interface.

To execute the tests, the parameters *discriminator*, *setup_code* and *pairing_mode* need to be filled in the device configuration parameters (*dut_config*).

To configure specific/custom parameters, please edit the project configuration to include the parameters in the session (*test_parameters*).

Project configuration example:
[source,xml,options="unbreakable"]
----
{
  ...
  "dut_config": {
    "discriminator": "3840",
    "setup_code": "20202021",
    "pairing_mode": "onnetwork",
    "chip_tool_timeout": null,
    "chip_tool_use_paa_certs": false
  },
  "test_parameters": {
    "paa-trust-store-path": "/credentials/development/paa_roots"
    "storage-path": "admin_storage.json"
  }
  ...
}
----

> Considering this project configuration and selecting tests TC_ACE_1_1 and TC_ACE_1_2. 

Reviewer: ```suggestion
Considering this project configuration and selecting tests TC_ACE_1_2 and TC_ACE_1_3.
```

image:images/img_61.png[]

This configuration will result in the following execution steps:
. Perform DUT commissioning

> . Run the TC_ACE_1_2 test (without commissioning)

Reviewer: ```suggestion
. Run the TC_ACE_1_2 test (with commissioning)
```

Author: In this case is correct. We ran the tc_ace_1.2 without commissioning...

> . Run the TC_ACE_1_3 test (without commissioning)

Reviewer 2: This is not being shown as a list

<img width="1073" alt="Screenshot 2023-12-14 at 15 54 19" src="https://github.com/project-chip/certification-tool/assets/116589288/e1546bd9-f53f-4eb0-a506-2e0e53f3fdd7">

Author: Done :)

 
 python3 TC_ACE_1_3.py --discriminator 3840 --passcode 20202021  --paa-trust-store-path /credentials/development/paa_roots --storage-path admin_storage.json

===== PIXIT Support

> PIXIT type parameters must be filled in in the section (*test_parameters*).

Reviewer 2: ```suggestion
PIXIT type parameters must be filled in the *test_parameters* section.
```

The following example will be used to define the following parameters:

----
PIXIT.ACE.APPENDPOINT:1
PIXIT.ACE.APPDEVTYPEID:256
PIXIT.ACE.APPCLUSTER:OnOff
PIXIT.ACE.APPATTRIBUTE:OnOff
----

Project configuration example:
[source,xml,options="unbreakable"]
----
{
  ...
  "test_parameters": {
    "paa-trust-store-path": "/credentials/development/paa_roots",
    "storage-path": "admin_storage.json",
    "int-arg": "PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256",
    "string-arg": "PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff"
  }
  ...
}
----

> The following example will be used to define the following parameters when running the test:

Reviewer 2: ```suggestion
The above example will be used to define the following arguments when running the test:
```
